core/build.gradle.kts

```diff
@@ -88,7 +88,7 @@ kotlin {
 
     }
 
-    js {
+    js(IR) {
```

Comment:
@FenstonSingel can I ask why you specified IR explicitly? Isn't the default IR now?
Just curious as in my libraries I'm now omitting it.

Reply:
Yes, it shouldn't be needed. Instead, the gradle property `kotlin.js.compiler=both` should be removed.